react/ReadMore/ReadMore.js

```diff
@@ -27,7 +27,8 @@ type Props = {|
   maxLines?: number,
   maxRows?: number,
   moreLabel: string,
-  lessLabel: string
+  lessLabel: string,
+  backgroundColor: 'card' | 'body'
```

Comment:
Then I believe we should have a variable for body background as well, so in case we decide to change the body background it stays connected

Reply:
Because the less implementation of this uses `@sk-background`, it will stay connected.